src/components/SaveButton.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react'
+import styled, { css } from 'react-emotion'
+
+const StyledSaveButton = styled('section')`
+  display: flex;
```

Comment:
Hier kannst du die `grid-area` einfügen aus dem Grid, das du in `App.js` angelegt hast.

Reply:
jupp